projects/storefrontstyles/scss/components/_icons.scss

```diff
@@ -0,0 +1,13 @@
+@import url('https://use.fontawesome.com/releases/v5.8.1/css/all.css');
```

Comment:
Just a question, can we/should we use this https://www.npmjs.com/package/@fortawesome/fontawesome-free instead of the cdn?

Reply:
I don't like another dependency, and I like a supper fast setup. 
I'm going to rework the styles structure, and will most likely come up with an approach to opt-out for the cdn download.